.babelrc

```diff
@@ -1,5 +1,12 @@
 {
-	"presets": ["env"],
+	"presets": [
+    ["env", {
+      "targets": {
+        "node": "8"
+      }
+    }]
+  ],
+	"plugins": ["transform-object-rest-spread"],
```

Comment:
node.green says this is available in node v8.3+: do we need a babel plugin?

Reply:
Ill changed the min version to 8.3+